### PR TITLE
Initial functional package

### PR DESCRIPTION
**Pull request by christopherDT**
The initial functional version of the package, with all main functionality present. To be beta tested while buttoning everything up for CRAN requirements

### CODE REVIEW

**Comment by christopherDT** [comment]
@dhhagan for review when available

File: LICENSE.md
Comment (dhhagan) [inline LICENSE.md:1]: Could we switch this to an Apache-2.0 license?

File: tests/testthat/setup.R
Comment (dhhagan) [inline tests/testthat/setup.R:1]: Does this mean that CRAN/R don't have a way to keep secrets out of test suites for the purpose of submitting to CRAN? I'm not familiar with the process, but am curious. Or is this something we can store in the github org as a secret and pull via an action to run tests/build/submit to CRAN?

File: R/wrappers.R [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+#' \dontrun{
+#'   get_data(sn = "MOD-PM-00808", limit = 100)
+#'   get_data(sn = "MOD-PM-00808", start = "2020-01-01 21:54", stop = "2023-03-03 05:47")
+#'   get_data(sn = "MOD-PM-00808", filter = "device_state,eq,ACTIVE")
+#'   get_data(sn = "MOD-PM-00808", sort = "timestamp,asc")
+#' }
+#'
+#' @returns The specified device data
+#' @export
+get_data <- function(sn, limit = NULL, start = NULL, stop = NULL, filter = NULL, sort = NULL, raw = FALSE){
```
Comment (dhhagan) [inline R/wrappers.R]: I think we may want to set a non-NULL limit as right now the default behavior is to grab all data for a sensor (I think?) which can easily be millions of records which isn't something we likely want to encourage.

**Review by dhhagan — CHANGES_REQUESTED** [review]
Just a few comments, but otherwise ready to merge!

**Comment by christopherDT** [comment]
@dhhagan -- both the license change and setting a `get_data()` limit default have been handled. Waiting to hear back from CRAN on how to handle the API key issue

**Review by dhhagan — APPROVED** [review]

**Pull request merged**